dashboard/app/helpers/profanity_helper.rb

```diff
@@ -11,7 +11,7 @@ module ProfanityHelper
   # @param [Integer] limit - Number of requests allowed over period.
   # @param [Integer] period - Period of time in seconds.
   def self.throttled_find_profanities(text, locale, id, limit, period)
-    return nil if text.nil_or_empty?
+    return yield(nil) if text.nil_or_empty?
```

Comment:
(this is the actual bug fix)

Reply:
What's the difference between returning nil and yield(nil) ?

Reply:
`yield(args)` will give `args` to a block and invoke that block (`yield` doesn't return, which is why i'm also returning here). this is how it's used:

https://github.com/code-dot-org/code-dot-org/blob/1533f07ca5d6b58294c4c1407990097fdca13ba1/dashboard/app/controllers/profanity_controller.rb#L29-L39

i wanted to make it so that this method accepted a block that is only executed if the request isn't throttled. if the request is throttled, you can ignore that or handle it (which i'm doing below by logging to HB).

in this particular case, i want to yield because i want the block to execute and return an empty response from `POST /profanity/find`